modules/services/sharing/src/main/kotlin/au/com/shiftyjelly/pocketcasts/sharing/SharingClient.kt

```diff
@@ -329,11 +344,13 @@ data class SharingRequest internal constructor(
 
         fun sharingTitle(): String
 
+        fun sharingSubject(): String
```

Comment:
I wonder if this is necessary. I think we could use `shringTitle()` for `EXTRA_SUBJECT. This property is copied in other types of `Data` anyway.